### PR TITLE
Update to v0.14.0-rc3

### PR DESCRIPTION
**Pull request by paluh**
Backlinking to purescript/purescript#3942

### CODE REVIEW

**Comment by JordanMartinez** [comment]
CI failed because `npm run -s test` failed. However, it seems that CI failed because there isn't a `test` script in `packages.json`. Do we add a `test` script that calls `pulp test`? Or something else?

**Comment by JordanMartinez** [comment]
@paluh Please update the `package.json` file to include the `test` script. CI will fail until we add this:
```json
  "scripts": {
    "test": "pulp test"
  },

```

**Review by JordanMartinez — APPROVED** [review]
CI builds now.

File: src/Data/Functor/Compose.purs [hunk 1]
```diff
@@ -17,6 +17,7 @@ import Data.TraversableWithIndex (class TraversableWithIndex, traverseWithIndex)
 import Data.Tuple (Tuple, curry)
 
 -- | `Compose f g` is the composition of the two functors `f` and `g`.
+newtype Compose ∷ ∀ k1 k2. (k2 → Type) → (k1 → k2) → k1 → Type
```
Comment (hdgarrood) [inline src/Data/Functor/Compose.purs]: Can you use ascii syntax here please?

**Comment by hdgarrood** [comment]
I’m happy to merge this once we’re using ascii syntax everywhere.

**Comment by paluh** [comment]
@hdgarrood Yeah, sure - once again sorry for this mess. I'm going to review all my 0.14 related PR's soon to ensure that I didn't make this mistake anywhere else.

**Comment by hdgarrood** [comment]
No worries at all ❤️ thanks again for your PRs!

**Comment by paluh** [comment]
`grep` claims that I don't have any more unicode syntax in this or any other 0.14 PRs. I hope that is true :-)

**Review by thomashoneyman — APPROVED** [review]

**Pull request merged**